src/Effects/Tremolo.js

```diff
@@ -0,0 +1,83 @@
+Pizzicato.Effects.Tremolo = function(options) {
+
+	this.options = {};
+	options = options || this.options;
+
+	var defaults = {
+		speed: 4,
+		mix: 0.8
+	};
+
+	// create nodes
+	this.inputNode = Pizzicato.context.createGain();
+	this.outputNode = Pizzicato.context.createGain();
+	this.dryGainNode = Pizzicato.context.createGain();
+	this.wetGainNode = Pizzicato.context.createGain();
+
+	this.LFOGainNode = Pizzicato.context.createGain();
```

Comment:
Acronyms in casing are [tricky](http://stackoverflow.com/questions/15526107/acronyms-in-camelcase). In Javascript, I prefer to have the acronym as if it was a regular word, so in this case ```lfoGainNode``` would be better than ```LFOGainNode```.